src/mito2/src/sst/version.rs

```diff
@@ -104,6 +106,25 @@ impl LevelMeta {
             files: HashMap::new(),
         }
     }
+
+    /// Returns expired SSTs from current level.
+    pub fn get_expired_files(&self, expire_time: &Timestamp) -> Vec<FileHandle> {
+        self.files
+            .iter()
+            .filter_map(|(_, v)| {
+                let (_, end) = v.time_range();
+                if &end < expire_time {
+                    Some(v.clone())
+                } else {
+                    None
+                }
+            })
```

Comment:
nitpick: You could use `values()` since you only need the file handle.
```suggestion
            .values()
            .filter(|v| {
                let (_, end) = v.time_range();
                end < *expire_time
            })
            .cloned()
```